tests/nvram_offsets.c

```diff
@@ -59,7 +59,13 @@ int main(void)
      * size of the OBJECT is the same on all architectures so that a full
      * NVRAM fits on all architectures
      */
-#define OBJECT_EXP_SIZE 1896
+#if RSA_4096
+# error Unsuported RSA key size
+#elif RSA_3072
+# define OBJECT_EXP_SIZE 2600
```

Comment:
Need to investigate why this increases by so much between RSA_2048 and RSA_3072.

Reply:
Parts of the OBJECT structure increase by 128, 320, and 256 bytes respectively, thus a total of 704 bytes increase (37%).